src/integrationtests/java/com/aws/greengrass/integrationtests/ipc/IPCServicesTest.java

```diff
@@ -136,6 +144,14 @@ void beforeEach(ExtensionContext context) {
         ignoreExceptionOfType(context, InterruptedException.class);
     }
 
+    @DisabledOnOs(OS.WINDOWS)
+    @Test
+    void Given_assign_path_for_ipcSocket_When_startUp_Then_ipcSocket_store_in_assigned_path() {
+        DeviceConfiguration deviceConfiguration = kernel.getContext().get(DeviceConfiguration.class);
+        String ipcPath = Coerce.toString(deviceConfiguration.getIpcSocketPath());
+        assertThat(new File(ipcPath), is(anExistingFileOrDirectory()));
+        assertEquals(ipcPath, Coerce.toString(kernel.getConfig().getRoot().lookup(SETENV_CONFIG_NAMESPACE, NUCLEUS_DOMAIN_SOCKET_FILEPATH_FOR_COMPONENT)));
```

Comment:
This tests if you're setting the path correctly for the client (inside the component) to use, what test is testing the server side that creates the socket and sets permissions etc?

Reply:
The test verifies that the socket exists. The subsequent tests in this class also exercise IPC and they will use the customized path because they are all using the same config file.

Reply:
Are there tests that also exercise the default behavior without customized path?

Reply:
all of the other tests in the nucleus that use IPC